### PR TITLE
Update grpc/grpc-js for deadline exceeded logging improvements

### PR DESCRIPTION
**Pull request by airhorns**
https://github.com/grpc/grpc-node/releases includes a number of bugfixes as well as some enhancements to the `RpcError: Deadline Exceeded` error message specifically. Lets use the new one!

### CODE REVIEW

**Comment by airhorns** [comment]
Tests are failing for the same reason they are failing on main I believe

**Comment by airhorns** [comment]
This is in better shape now!

File: js/src/grpc-client.ts [hunk 1]
```diff
@@ -337,10 +328,9 @@ export class DateiLagerGrpcClient {
 
   /**
    * Update an object.
-   *
    * @param project The id of the project.
    * @param obj     The object to update.
-   * @returns         The latest project version or `null` if something went wrong.
+   * @returns       The latest project version or `null` if something went wrong.
```
Comment (angelini) [inline js/src/grpc-client.ts:333]: I have no idea why but my and Scott's auto formatter was setting it up this way.

File: js/src/utils/errors.ts [hunk 2]
```diff
@@ -1 +1,4 @@
+/**
+ *
+ */
```
Comment (angelini) [inline js/src/utils/errors.ts:3]: ?

**Review by angelini — APPROVED** [review]

**Pull request merged**